debian/postinst

```diff
@@ -14,8 +14,9 @@ export_gpg_key_from_shared_keyring() {
     KEY_ID=$1
     if [ ! -z "$KEY_ID" ]; then
         rm -f $ESM_APT_GPG_KEY
-        gpg --no-auto-check-trustdb --options /dev/null --no-default-keyring \
-            --keyring $UA_KEYRING_FILE --export $KEY_ID > $ESM_APT_GPG_KEY
+        gpg --no-auto-check-trustdb --no-options --no-default-keyring \
+            --keyring $UA_KEYRING_FILE --export $KEY_ID \
+	    --output $ESM_APT_GPG_KEY --yes
```

Comment:
Indentation here looks off.

Reply:
Which part of this change is it that stops the creation of the .gnupg dir?

Reply:
@OddBloke fixed indentation. and --no-options param as mentioned in the commit msg is the one that stops the ./gnupg dir creation :)

Reply:
Aha, right, I should have read the _entire_ manpage entry:

> Shortcut for --options /dev/null. This option is detected before an attempt to open an option file.  Using this option will also prevent the creation of a ‘~/.gnupg’ homedir.

I read the first sentence and was like "there's no change here", oops!